app/views/open_studios_subdomain/_container_header.slim

```diff
@@ -0,0 +1,6 @@
+.container-header
+  a.container-header__brand href=root_path title="Mission Artists"
```

Comment:
is this root_path the open studios domain?  I guess it must be because it's "/" and we're in the subdomain.

cool!